permissions/resource_permissions.go

```diff
@@ -203,6 +203,16 @@ func (a PermissionsAPI) Delete(objectID string) error {
 // Read gets all relevant permissions for the object, including inherited ones
 func (a PermissionsAPI) Read(objectID string) (objectACL ObjectACL, err error) {
 	err = a.client.Get(a.context, urlPathForObjectID(objectID), nil, &objectACL)
+	apiErr, ok := err.(common.APIError)
+	// https://github.com/databrickslabs/terraform-provider-databricks/issues/1227
+	// platform propagates INVALID_STATE error for auto-purged clusters in 
+	// the permissions api. this adds "a logical fix" also here, not to introduce
+	// cross-package dependency on "clusters".
+	if ok && strings.Contains(apiErr.Message, "Cannot access cluster") {
```

Comment:
Also test for `apiErr.StatusCode`? I expect it to be stable for this error message.

Reply:
fixed in #1253